pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -0,0 +1,233 @@
+
+# support list files routine
+# use the default pysat method
+dname = '{year:02d}{month:02d}{day:02d}'
+vname = '.{version:03d}'
+supported_tags = {ss: {'vtec': "gps{:s}g{:s}.hdf5".format(dname, vname)}
+                  for ss in inst_ids.keys()}
+list_files = functools.partial(ps_gen.list_files,
+                               supported_tags=supported_tags,
+                               two_digit_year_break=99)
+
+# madrigal tags
+madrigal_inst_code = 8000
+madrigal_tag = {'': {'vtec': 3500}}  # , 'los': 3505}}
+
+# support listing files currently available on remote server (Madrigal)
+list_remote_files = functools.partial(mad_meth.list_remote_files,
+                                      supported_tags=supported_tags,
+                                      inst_code=madrigal_inst_code)
+
+
+def init(self):
+    """Initializes the Instrument object with values specific to GNSS TEC
+
+    Runs once upon instantiation.
+
+    """
+
+    ackn_str = ''.join(["GPS TEC data products and access through the ",
+                        "Madrigal distributed data system are provided to ",
+                        "the community by the Massachusetts Institute of ",
+                        "Technology under support from U.S. National Science",
+                        " Foundation grant AGS-1242204. Data for the TEC ",
+                        "processing is provided by the following ",
+                        "organizations: UNAVCO, Scripps Orbit and Permanent",
+                        " Array Center, Institut Geographique National, ",
+                        "France, International GNSS Service, The Crustal ",
+                        "Dynamics Data Information System (CDDIS), National ",
+                        " Geodetic Survey, Instituto Brasileiro de Geografia",
+                        "e Estatística, RAMSAC CORS of Instituto Geográfico",
+                        " Nacional del la República Agentina, Arecibo ",
+                        "Observatory, Low-Latitude Ionospheric Sensor ",
+                        "Network (LISN), Topcon Positioning Systems, Inc., ",
+                        "Canadian High Arctic Ionospheric Network, ",
+                        "Institute of Geology and Geophysics, Chinese ",
+                        "Academy of Sciences, China Meterorology ",
+                        "Administration, Centro di Ricerche Sismogiche, ",
+                        "Système d’Observation du Niveau des Eaux Littorales",
+                        " (SONEL), RENAG : REseau NAtional GPS permanent, ",
+                        "and GeoNet—the official source of geological ",
+                        "hazard information for New Zealand.\n",
+                        mad_meth.cedar_rules()])
+
+    logger.info(ackn_str)
+    self.acknowledgements = ackn_str
+    self.references = "Rideout and Coster (2006) doi:10.1007/s10291-006-0029-5"
+
+    return
+
+
+def download(date_array, tag='', inst_id='', data_path=None, user=None,
```

Comment:
I downloaded a few days for testing but the files I got are labeled `stuff.hdf`. The load time is really quick though, suggesting they are actually netcdf files. Poking at the code the 'netcdf' path is indeed being run.

Reply:
yeah, regardless of the format you ask for and get, the remote files are listed as .hdf5.  Not sure if I fixed the rename here or in the other PR.